coreconfig/config.go

```diff
@@ -318,7 +319,7 @@ var getTimezone = func() (timezone string, err error) {
 // setTimezone sets the specified timezone for the system, an error is returned
 // if it can't.
 var setTimezone = func(timezone string) error {
-	if err := helpers.CopyFile(filepath.Join(tzZoneInfoPath, timezone), tzZoneInfoTarget, helpers.CopyFlagOverwrite); err != nil {
+	if err := cp.CopyFile(filepath.Join(tzZoneInfoPath, timezone), tzZoneInfoTarget, cp.CopyFlagOverwrite); err != nil {
```

Comment:
cp seems slightly on the opposite end: too specific for being a package. It'll be hard for us to find anything else quite fitting here. I suggest having this inside osutil for the time being, with a plan B of moving it into "fsutil" if we end up having too many filesystem manipulation routines inside osutil.